js/components/portal-dashboard/feedback/feedback-legend.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import AwaitingFeedbackActivityBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-activity-badge-icon.svg";
+import GivenFeedbackActivityBadgeIcon from "../../../../img/svg-icons/given-feedback-activity-badge-icon.svg";
+import AwaitingFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/awaiting-feedback-question-badge-icon.svg";
+import GivenFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/given-feedback-question-badge-icon.svg";
+import UpdateFeedbackQuestionBadgeIcon from "../../../../img/svg-icons/update-feedback-question-badge-icon.svg";
+import { FeedbackLevel } from "../../../util/misc";
+
+import css from "../../../../css/portal-dashboard/feedback/feedback-legend.less";
+
+interface IProps {
+  feedbackLevel: FeedbackLevel;
+}
+
+export const FeedbackLegend: React.FC<IProps> = (props) => {
+  const {feedbackLevel} = props;
+  const awaitingFeedbackIcon = feedbackLevel === "Activity" ? <AwaitingFeedbackActivityBadgeIcon /> : <AwaitingFeedbackQuestionBadgeIcon />;
+  const givenFeedbackIcon = feedbackLevel === "Activity" ? <GivenFeedbackActivityBadgeIcon /> : <GivenFeedbackQuestionBadgeIcon />;
```

Comment:
You might consider breaking the above two lines onto multiple lines as they're getting pretty long.  I know this project doesn't seem to have a lint rule for line length, but at some point we informally starting breaking lines that ran over 130 characters (which is what our other projects use).

Reply:
So I would do like:
```
  const awaitingFeedbackIcon = feedbackLevel === "Activity" 
                               ? <AwaitingFeedbackActivityBadgeIcon /> 
                               : <AwaitingFeedbackQuestionBadgeIcon />;
                             
```

Reply:
And yes, there are lots of modules that violate this rule.  It's a non trivial task to fix them all or add the lint rule so my current mindset has been to try to fix long lines whenever I'm working on code in the area.